src/components/card/FlipCard.js

```diff
@@ -0,0 +1,140 @@
+import styled, { css } from 'styled-components/macro'
+import { useState } from 'react'
+import mainBackground from '../../assets/images/mainBackground.jpg'
+
+export default function FlipCard(props) {
+  const [isCardFlipped, setIsCardFlipped] = useState(false)
+  const isFreeOfCharge = props.freeOfCharge
+  const handleFlipCardClick = () => {
+    setIsCardFlipped(!isCardFlipped)
+  }
+  return (
+    <Card onClick={handleFlipCardClick}>
+      <CardContent isCardFlipped={isCardFlipped}>
+        <CardFront>
+          <CardTitle>{props.name}</CardTitle>
+          <CardSubtitle>{props.city}</CardSubtitle>
+        </CardFront>
+        <CardBack>
+          <CardBody>
+            <p>
+              <b>Name:</b> {props.name}
+            </p>
+            <br />
```

Comment:
instead of the `<br />` you could set a margin bottom in the card body:
```
const CardBody = styled.div`
  transform: translateZ(6rem);
  font-weight: 400;
  font-size: 0.9rem;
  line-height: 1.6;
   p {
      margin-bottom: 1rem;
   }
`
```